### PR TITLE
Add processed fiber photometry

### PR DESCRIPTION
**Pull request by weiglszonja**
Add data from processed fiber photometry

TODO:
- [x] Add raw fiber photometry series to acquisition
- [x] Add processed data to ophys
- [x] Figure out single vs multi experiment type (do not write second fiber metadata for single fiber experiment)
- [x] Add velocity and acceleration (continuous)
- [x] Add reward events
- [x] Add movement events (movement onset, acceleration and deceleration onset)
- [x] Add peak transient events (the onset of large transient peaks for green or red fiber fluorescence)

### CODE REVIEW

**Comment by weiglszonja** [comment]
Full [here](https://us-east-2.console.aws.amazon.com/s3/object/catalystneuro-processing?region=us-east-2&bucketType=general&prefix=dombeck-lab-to-nwb/20200129-0002_ver2.nwb)

File: src/dombeck_lab_to_nwb/azcorra2023/azcorra2023_metadata.yaml [hunk 1]
```diff
@@ -13,18 +13,30 @@ NWBFile:
...
     - Azcorra, Maite
-
-Ecephys:
-  ElectricalSeriesVelocity:
-    description: The electrical series of the velocity from rotary encoder collected at 4000 Hz by Picoscope.
-  ElectricalSeriesFluorescenceRed:
-    description: The electrical series of fluorescence from Red channel collected at 4000 Hz by Picoscope.
-  ElectricalSeriesFluorescenceGreen:
-    description: The electrical series of fluorescence from Green channel collected at 4000 Hz by Picoscope.
+  virus: |
```
Comment (alessandratrapani) [inline src/dombeck_lab_to_nwb/azcorra2023/azcorra2023_metadata.yaml:27]: I would put this description under pharmacology and use the field virus for reporting just the virus notation (AAV8-EF1α-CreOn/FlpOn-GCaMP6f (RRID:Addgene_137122, titer 6.10 × 1013)
    for Aldh1a1-iCre/Th-Flpo, Vglut2-IRES-Cre/Th-Flpo and Calb1-IRES2-Cre/Th-Flpo mice or AAV1-CAG-FLEX-GCaMP6f (RRID:Addgene_100835, titer 2.00 × 1013) for DAT-Cre or Anxa1-iCre mice)

**Review by alessandratrapani — APPROVED** [review]
looks good

File: src/dombeck_lab_to_nwb/azcorra2023/interfaces/azcorra2023_fiberphotometryinterface.py [hunk 2]
```diff
@@ -92,9 +77,18 @@ def add_to_nwbfile(
...
+            for trace in traces_metadata
+            if trace["name"] in channel_name_to_photometry_series_name_mapping.values()
+        ]
 
         excitation_sources_metadata = fiber_photometry_metadata["ExcitationSources"]
+        excitation_source_ind = set([trace["excitation_source"] for trace in traces_metadata_to_add])
+        excitation_sources_to_add = [excitation_sources_metadata[ind] for ind in excitation_source_ind]
+
         excitation_sources_description = (
             "Blue excitation light (470 nm LED, Thorlabs, M70F3) and purple excitation light (for the isosbestic "
```
[the remark this answers is not in the document]
Comment (weiglszonja) [reply src/dombeck_lab_to_nwb/azcorra2023/interfaces/azcorra2023_fiberphotometryinterface.py:93]: sure I'll change this, just copied this from the manuscript I think.

**Comment by weiglszonja** [comment]
Thanks, I'll add a follow up PR to switch to `ndx-fiber-photometry` soon.

**Pull request merged**